.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug with the specification
+
+---
+
+<!--
+Thank you for reporting a possible bug in ECMA-262.
+
+If this is a bug with a specific implementation of ECMA-262, such as
+a JavaScript engine or a browser, please report it on their bug tracker
+instead.
+
+Please provide code that demonstrates the problem, keeping it as
```

Comment:
For spec bug reports, it may or may not be meaningful to include JS code. In fact, I'm not really sure what guidance to give bug report authors beyond the paragraph above. Thoughts?

Reply:
fair point. I'd be fine removing it. let's see what others think

Reply:
What would be awesome is walking them through providing `eshost` results, maybe?

Reply:
@mathiasbynens @bterlson can you verify the jsvu/eshost setup steps

Reply:
Left a comment. +1 to suggesting including an `eshost` command + output, as this often makes it easier to understand what's going on. For those bug reports where that doesn't make sense, people can omit it.

Reply:
LGTM!